client/src/views/RegisterPage.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="h-screen w-screen flex-col bg-gradient-to-br from-cyan-400 to-green-500 pt-10 font-header">
+    <div class="ml-auto mr-auto h-1/6 w-1/2 items-center justify-center rounded-3xl bg-yellow-50">
+      <p class="pt-8 text-4xl font-semibold leading-10 phone:pt-10 phone:text-xl">Welcome to your To-do App</p>
+    </div>
+    <div class="h-2/55 m-auto mt-10 justify-center rounded-3xl bg-yellow-50 phone:w-8/12 computer:w-4/12">
+      <p class="ml-auto mr-auto w-full pt-4 text-3xl font-semibold">Register</p>
+
+      <UserInput :label="'First Name:'" :property="'firstName'" v-model="user.firstName" />
```

Comment:
if you just want to pass a string as prop you don't need to use bind ( : ) so instead of :label="'First Name'" you can do label="First Name"